authentication/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "oauth",
+  "version": "1.0.0",
+  "description": "server.js",
+  "main": ".mongorc.js",
+  "dependencies": {
+    "body-parser": "latest",
```

Comment:
Where is `cookie-parser/express-session/read-files-promise/morgan` module required?